hr_attendance_management/models/hr_employee.py

```diff
@@ -14,18 +18,27 @@ class HrEmployee(models.Model):
     ##########################################################################
     #                                 FIELDS                                 #
     ##########################################################################
-    extra_hours = fields.Float(compute='_compute_extra_hours',
-                               store=True)
-    extra_hours_lost = fields.Float(compute='_compute_extra_hours_lost',
-                                    store=True)
+    extra_hours_status = fields.Selection([(True, 'annual'),
```

Comment:
A help parameter describing the purpose of this field would be welcome. Should it be a Boolean field instead with maybe a better name? Or why do you make a Selection? I think in any case that it will be hard for people that don't know the module to understand what is this doing.